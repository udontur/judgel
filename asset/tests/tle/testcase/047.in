3568 169
64300 41545 79472 32359 71867 23508 14228 15800 32239 55171 50906 26470 57160 66184 84523 27682 4187 81435 1432 26496 9003 54897 3205 82556 54013 34613 69276 67646 18613 16767 40380 19037 71114 36387 23105 86785 42858 19656 13019 161 9407 70301 8469 66865 74787 3785 16200 46218 26578 36886 28981 10378 73152 72057 64503 30580 28457 82645 91859 9599 80670 2230 78315 6904 91865 29670 14633 26450 96332 95502 73079 17507 24310 63630 69268 56742 30320 74411 6715 97222 2232 40709 94392 53789 43400 7437 73721 45244 52036 33936 21082 91032 92650 62698 68712 78041 14017 22227 97375 91082 4151 15397 5018 48319 37323 5111 5477 15315 39950 63699 9186 8374 43277 96444 13661 22856 86681 41158 43805 6559 23685 54732 80185 34201 67631 86407 40435 39892 6019 58288 24 62290 85590 41387 91659 11415 36585 39855 21157 54752 94324 78816 42773 9413 50921 28595 78812 93647 41417 57 85178 52458 51153 12364 18023 59973 59206 22140 73378 49978 16053 26469 61050 93536 61441 74683 36452 8405 40488 27289 46871 6203 78056 12910 89760 51692 19013 68434 97724 4101 37594 47273 85955 53188 16702 32902 82256 1929 14794 84044 3801 21139 99532 84118 89724 443 64608 35820 65627 59488 88941 55284 41073 19943 32706 21403 160 91309 41990 10216 76422 99477 16676 18171 93320 72331 30264 12269 16111 98733 33485 55786 98939 96707 38487 87671 96195 58046 30786 36006 63349 25903 3240 92979 54185 48224 28743 21307 66196 36819 88199 31015 92209 90059 50378 41373 17414 83131 95824 98532 56909 85465 8113 29001 30169 92196 3758 98428 85016 79920 8051 47914 73191 79380 60643 9682 58308 42017 96726 9030 90478 83111 29602 58503 838 27644 73962 37601 34995 38353 52387 22667 94055 24276 51415 26574 86904 74052 46221 69256 33561 83918 69477 15756 84452 52848 37064 3032 80645 22855 65776 33509 16092 76836 51714 65045 33305 68252 61369 422 42088 44311 48484 42045 52851 87344 87882 55567 69654 82170 30635 43377 63170 24285 92627 85940 79870 19671 84513 1870 22427 71946 94302 38512 99231 7834 77457 91745 29150 6005 45890 34950 53353 9884 26816 91619 59201 35080 8089 82612 67826 19906 57010 95359 22251 67878 8108 62982 43022 10888 82421 496 33000 87524 40994 73180 37356 71881 18313 6192 99595 52067 98810 75083 96451 37177 63119 40995 38081 19452 2112 21531 95182 87183 91473 46125 1353 12662 42141 4872 49533 10110 77927 16532 59586 28797 53989 18408 71538 99969 62043 94069 59178 42673 67920 53808 12552 85194 65562 20663 37181 86774 18905 46481 62225 46548 46714 44694 17541 87444 73170 87297 54179 97157 88786 90661 90273 58212 84812 52984 49756 28196 42936 5261 55906 68588 10215 14544 24760 67239 15060 27613 39473 73331 65642 72473 65310 70799 39887 79627 52497 12521 96424 83543 52244 65923 73721 62006 75276 44622 28219 53628 69196 83736 10138 38732 70235 13839 33338 27152 67883 4556 85183 97985 70822 15861 99947 1847 42355 9424 88771 38753 21527 21940 34817 40737 61633 98242 13364 22093 718 42220 35556 15272 41212 30515 22934 64961 88015 359 39249 71707 45191 19137 15074 62172 49427 57977 97777 38780 10708 15357 68039 83991 55181 99757 88688 76566 78516 63558 45149 44057 25664 92061 19370 77339 91 12179 79 80479 52765 58653 40604 39160 88250 72795 18872 25303 50735 9740 64014 91881 83899 84058 80351 75383 63308 15787 46688 55281 37641 42228 80033 38321 51760 19288 95809 33405 71893 80961 7259 22071 1085 99008 55400 70504 64935 10883 64772 74051 66062 27599 57881 49153 40226 1584 45133 47064 29085 18365 58132 66605 45222 84441 23104 26188 83471 46669 95918 62575 1395 99188 46841 50245 14934 62767 25096 3593 12124 6139 87055 63421 35090 10041 67962 47376 52866 69735 67650 40573 67949 67468 24375 48844 11082 68166 86738 78901 23664 92510 13442 98835 78700 36105 71459 36751 80375 3939 96918 71126 41188 39541 29918 31629 48573 20753 66683 32686 41797 46077 84776 38352 25250 18779 23547 67552 72944 4194 51459 8845 19850 70748 11511 84849 78535 18323 94783 99434 73757 55636 63867 64307 45484 8252 81524 77778 74777 79236 13312 84722 28224 14129 28238 67197 40706 14253 82103 24418 44188 79809 6568 61826 83564 67394 90630 31676 458 16475 8748 32818 30497 93419 11639 6977 44990 57742 60759 51909 25635 51504 86525 82018 30779 72610 89970 77034 27116 24721 60612 60905 67979 2344 413 20640 90383 38775 23903 84521 98488 7384 69280 13468 25716 14908 81931 55478 47641 38886 92318 30297 45402 95602 96700 12910 9248 64290 43732 93476 61375 57759 19337 79737 16754 56415 69544 29172 74992 17554 99954 13096 49206 10667 29807 68860 19384 37869 13926 2578 47458 98572 79601 37824 60711 16589 16737 41029 4820 16692 14270 98249 19318 19554 75146 57076 6888 35131 46894 89737 98456 52134 40485 6549 86284 57727 80049 76815 28924 55126 86840 20252 72810 46508 91699 76269 96502 48781 16869 8495 48058 56557 43193 39435 46740 49304 41181 69470 91528 86389 10922 21966 56035 75942 75907 42471 40058 50806 20510 45881 71709 99815 55415 14076 94934 21123 92003 84491 46159 8698 33203 51163 23617 7730 68557 20449 8630 53219 7286 53349 26529 61086 45967 5710 84866 36857 4883 68568 28267 47752 21548 95167 15109 4317 39423 99257 88037 17153 10931 9489 4969 92957 17466 39202 13335 68865 68695 48286 13352 94791 41256 80346 35944 14584 53597 26623 6575 31256 90463 49260 89976 86824 58961 86157 29476 80317 97580 74079 26255 60617 1783 93696 23551 2358 32281 70525 43811 62435 94407 3244 37275 45253 14584 67294 19874 12912 95628 10392 68872 45831 82590 94420 6301 44825 13040 33002 50125 57209 90460 56698 46518 44935 40883 92115 31898 25392 65268 66640 1338 22689 22907 38225 17509 10545 60575 10896 52844 42154 49076 47197 68988 11375 681 10727 87774 15805 55393 3487 84335 8663 192 77210 46692 18836 39655 53524 53131 81312 29566 23111 48614 52618 60707 99862 14656 7193 83918 41221 88547 76402 41029 17758 27809 23302 54443 27253 64355 56766 32211 82649 84697 13084 71872 6983 63113 16643 19128 20007 77902 76255 34851 83403 15447 914 47668 14907 14930 5353 80407 21012 39121 72920 32234 41166 95448 21676 57532 72150 71424 11968 86255 20637 55360 68477 68027 51130 75795 93564 51981 70655 73096 66341 48857 81916 77665 46270 92821 3427 76322 17201 32391 79763 48502 26784 16066 28847 33653 15037 12415 45607 30885 41405 93928 95022 45630 92191 54683 89865 27384 9015 24177 96515 73116 81001 74766 48048 29783 42682 49495 35509 47549 94815 72681 81437 12179 26733 58653 97558 49097 97661 69651 92118 19637 11161 61819 94954 96963 24636 41227 35161 70101 83890 61432 49121 67616 85038 81280 34448 17557 63917 43774 94971 45675 97497 61187 85126 74589 64440 11622 1032 70792 17812 22150 71660 58301 56731 27837 56672 39252 11549 87640 66293 75474 12991 13025 51678 95336 81723 6129 61780 27048 64694 82411 98104 41640 42188 76006 87714 31261 14627 50920 80830 24271 74498 89080 26175 50188 16777 32189 61378 37271 73013 18893 76775 21111 10835 21904 67103 91746 22022 26188 49718 26501 33356 59042 8019 42112 4402 99707 93119 42669 64043 96445 4647 92893 80109 60594 42217 49432 1284 38870 92614 64432 53461 55742 14769 36756 77274 50685 37725 32430 11066 47826 80167 47252 47011 59563 32794 8642 60637 68327 42570 96961 66646 86030 83694 89989 94392 54570 89058 82827 15242 48593 67467 72463 94853 49049 44483 96641 65807 18477 15930 38116 76162 16230 22063 28692 86965 73038 70808 18773 32713 55373 81429 27456 65635 28931 14540 71771 74616 71799 78599 98316 18990 28405 44681 77050 45930 66561 59542 56175 56418 91968 19693 98086 2506 66125 20572 74563 11951 17507 48447 52465 17234 94389 55185 94404 49578 30751 2461 62642 24454 61868 45604 59036 31040 60043 72890 44514 8311 65142 47357 89654 35669 93678 37289 34164 56113 10221 75512 56097 79017 74515 91937 46241 98814 55529 83467 23849 26964 12417 29837 720 27256 55083 59845 77306 91830 21168 53953 60227 28855 10604 11078 28099 55020 35561 67989 45608 21704 2718 35228 80632 32566 32615 67311 48305 55945 69655 58278 40507 86677 42791 48242 20229 55653 11104 98087 66338 67550 57397 30308 93730 38597 67467 84200 47434 28637 32019 1349 65001 89894 16434 79933 19635 25567 70514 60909 4616 57909 56046 12640 13971 40871 27586 78035 61754 87233 95561 7882 50956 37689 16422 44827 80077 57002 56030 81099 21372 88410 7643 54419 72684 854 4373 81879 49240 57053 38725 57086 4809 10256 84227 837 79982 93343 32415 46952 65792 70653 60695 55818 26307 36061 99720 37173 91486 64331 23603 2460 91364 88762 19080 96963 68182 88568 37879 3086 73495 5668 86340 10610 20902 81227 12188 55930 79470 38020 85266 32784 80082 55111 37737 67341 410 58290 255 27621 23938 40230 65215 18673 3597 76009 35786 77908 72001 33804 64790 97237 7243 33004 88082 1212 21514 80794 73407 23101 67752 89219 76046 94336 52904 6588 67083 63147 9968 57518 95927 46638 75618 570 5204 50308 37454 32109 73461 46215 82303 83663 49481 25407 18102 39868 9579 65536 79433 73900 97394 43466 49107 12507 753 11227 69093 96132 44009 64489 30352 50748 69914 21264 46180 98114 35752 68762 96954 63770 34949 81926 73871 76534 54790 9519 22755 93545 23255 75742 4457 60201 46767 16180 45040 73441 67187 1626 52358 77864 13495 34333 52394 62995 38407 61877 59886 24724 64900 21172 22904 93100 66264 1609 37598 13912 26493 34671 33910 32711 38777 57721 99791 53402 6178 33265 39551 24943 39366 59303 31511 22628 33380 98498 58395 40561 75372 51297 91930 79947 10740 44935 74814 46702 62975 25400 61702 54810 86755 79866 77627 72669 20943 41287 1022 55623 90025 36698 28116 30477 74135 48593 88530 88120 78529 55587 27907 54878 95095 9765 58135 19826 94911 39720 82968 63188 40874 60658 11737 4681 56572 76256 58753 52757 89293 6323 57284 55755 38969 78999 655 460 50581 20669 76310 28558 46111 85968 58936 92351 8325 15338 95156 45904 66444 43585 49854 17006 34855 42503 53415 59628 32310 67257 8670 98479 3498 60222 40002 7623 90272 35691 44546 43093 10662 67916 32302 97619 75336 81387 74305 58005 61089 99712 99780 13524 23244 11421 44786 96465 87529 52962 5962 64530 87398 77519 5889 10251 86958 54754 400 62560 63484 61175 909 66527 31975 17097 61986 59681 99237 91575 74174 19236 63174 5640 27121 77677 22538 10627 88459 72049 8164 35724 45366 14856 83955 3615 60948 37652 2558 71321 62233 20378 28024 8146 55325 68934 42332 6877 91595 64441 18180 99718 63732 41270 20767 39349 95153 66464 93887 18150 8566 14844 16741 95978 65572 67413 46931 77125 64886 46679 70774 21793 61884 66953 84752 37909 54431 38086 96583 91321 97522 32365 83059 73536 61129 98762 63925 21682 73181 62461 28196 58281 79734 8470 44152 68344 66646 7260 2219 56010 66744 518 19344 74419 12140 8872 15569 66585 75900 43229 52665 4591 91631 17106 27039 72769 99221 38137 79001 4588 13926 22854 28293 15662 60907 94348 21654 68807 22579 86250 6377 89576 60610 45826 93458 36918 9734 83633 34076 6947 95422 12395 38194 7919 91662 17111 69121 42544 86925 15830 97939 625 34894 13211 82732 72071 95238 17063 48017 45372 7204 88752 43446 59125 42432 51047 36230 7713 82584 84450 43159 86425 6116 61899 84828 50951 70559 93074 59931 70056 21790 41794 1466 91931 271 11246 76062 46732 4457 74398 26280 12135 47166 22903 37365 95761 76032 81 42675 23819 95704 80970 40420 45851 85029 64594 63461 92690 99330 51870 52164 54117 70184 92860 10803 6368 1430 14861 7111 67378 7530 42487 92217 97880 11223 54026 77626 49311 39310 29526 22216 93788 27641 34512 66467 33443 97969 50564 9564 13950 68334 73209 90190 9427 61333 26972 87971 47901 21982 90603 90550 93082 10064 51488 90011 13817 24904 55271 958 74007 52972 86489 19054 88854 21988 72818 75132 88841 60157 13332 84897 11158 37833 2085 15398 51409 61415 44303 19402 61469 1874 82985 83283 46460 9678 46345 29884 67733 88612 30071 79928 71652 73344 37173 27009 39772 96478 63100 82293 63695 43529 22219 83858 77173 12388 57845 30291 87966 4842 67398 58609 44013 67282 97637 59390 3573 70168 51794 59143 11063 18072 87052 81205 54808 97684 20530 67978 41797 39525 29504 27196 49814 8046 86227 69912 17892 2880 77029 83818 21604 41095 87100 5577 46207 12811 82263 52447 46590 51104 46012 78216 64751 6492 34931 29740 25971 31706 2734 47338 33395 49461 50659 42585 55674 74264 40271 52991 11630 27316 46729 22921 63074 75172 36817 30597 20781 8532 2659 99104 22822 61659 80796 72696 68121 66102 41974 93279 99348 34248 88579 35346 18036 12483 97937 17356 31803 68614 93837 87698 15092 70317 21228 59589 72691 78940 81392 94588 84643 44438 16909 83507 31264 89157 77562 4716 25315 15496 54142 28546 58985 75041 70824 5985 67162 64109 43937 15928 34702 68665 31377 83019 92264 72960 63162 89570 67498 59854 11790 62188 53107 13095 14606 63027 23276 83875 45355 62354 49203 71356 54012 29848 67129 58124 31200 60141 71049 46679 37979 63586 89355 21651 96028 28856 27940 20548 69512 34249 49229 92503 45056 1344 45347 7916 44393 23342 72076 93717 3234 34473 27084 45624 51448 16693 89133 46990 1135 28171 25676 27276 37053 41351 97127 83677 62548 65222 92301 69184 20176 12119 805 3958 78023 40851 1693 36615 18562 4839 7684 9642 690 8194 74144 12648 50382 23226 95376 46883 38345 29914 43449 65407 26214 24966 26684 20457 55427 6759 83390 62648 90482 4400 4586 82436 88755 34687 93713 47831 81145 16807 69326 54404 81893 65803 11830 77566 6987 79320 11030 75538 6337 81329 89499 23713 78091 16264 25527 51410 2653 27745 52948 55043 98031 81104 26989 55084 13162 45581 25219 45140 33912 61169 45162 88458 2077 3558 5968 41469 84004 52406 34231 15555 22029 51237 83284 16664 91405 85392 59050 47955 8498 97977 6650 95305 57252 67647 37568 24207 20100 62176 9703 42323 60229 87824 69939 21651 26662 48264 11076 82073 53617 53941 54586 78737 70133 59932 47784 15903 54559 955 96747 10405 85897 73766 3774 77212 16177 40228 6913 35936 64967 77491 52884 97972 1045 39575 76581 58380 5720 74498 19873 64323 48538 19333 45355 29418 37703 18886 65542 8844 59192 78460 47181 48579 15738 39062 37485 9281 25612 92711 54942 6175 37752 88974 67491 75615 97727 51311 2117 96143 5822 84486 73286 32936 13495 81863 57571 45459 18511 4173 45727 30813 5647 43171 76396 90544 31124 88507 44122 94651 72444 51114 44425 85286 91220 73525 6829 25682 93820 1399 53120 47016 88424 68107 20504 99150 31222 11244 50439 29978 57773 71156 15939 59260 49368 9537 70874 85933 75390 6309 99643 90438 55454 89446 92296 27006 15013 47333 76956 51841 92681 15505 17235 19377 30521 93529 87566 54398 84985 917 30065 48340 22911 62477 31605 87083 53866 7440 47399 37028 44618 29183 59401 26487 24710 57962 27086 69215 29680 4268 99667 31676 24792 47728 52810 3565 22335 60719 9451 64412 4061 61684 49750 40495 23093 86978 41831 38201 94346 20490 68020 9074 44557 17103 78698 86280 43150 6970 74536 57202 81258 50071 87657 69692 29648 5673 89835 98437 27289 18834 15213 95938 82699 70895 87479 81634 32168 42085 82940 74380 83570 60037 13550 79111 8703 40975 17727 53002 32071 72560 72507 19937 21500 23865 61295 92621 57114 80376 68236 42357 87086 83795 67334 25528 30997 95435 94827 27329 99592 85534 76066 41677 53258 86980 18929 58648 69481 37262 18271 47219 67340 62356 78586 17421 93945 44024 43891 1865 15029 74662 80081 83832 6012 17676 58920 15052 73569 77831 70959 479 9027 16613 66119 39314 40474 35887 96248 54275 21889 10274 13760 14746 59456 52959 4869 8045 64084 52979 2427 75303 53288 28460 31295 18440 76814 65794 77812 35467 22112 32896 8640 49226 45985 920 54209 5701 23520 47471 49594 61048 20147 59743 94908 90010 78998 29414 6847 99888 9290 36560 33366 10852 97812 16208 80683 15039 81927 87737 13573 32196 84531 64379 42602 19311 77033 9281 91331 65339 10802 13346 45746 44750 65420 31030 36845 13164 82608 74242 51716 23074 78551 62770 5171 5965 56704 58601 11848 54524 70598 55129 42877 46994 20678 52890 14321 29245 12541 21111 65781 80259 78895 41035 96819 73677 22074 6420 72653 75433 51579 70843 16319 24716 83980 38073 60402 7095 80690 6390 76521 32787 94979 32935 48830 36555 42341 80545 98237 28030 27196 11799 61179 31124 61554 71499 60232 48216 95407 73659 27208 32726 82521 43706 79970 36708 51329 84415 90022 8667 59295 62134 32316 56595 37009 22825 88717 19172 32006 15174 34080 37624 96474 61702 54481 10748 36585 97051 99555 76876 20426 8397 34184 68139 80145 91509 74393 71680 69363 23443 75810 46565 47845 24211 59933 62440 55975 48858 54662 75739 84058 26065 64973 56585 67004 21041 7134 27329 98938 78764 30336 77925 33351 23588 33403 68812 43938 42634 7807 1169 56808 33326 17320 32504 97991 87535 8603 46082 53409 62523 8263 66694 12398 99110 84138 12361 31639 85296 77922 43486 67757 9803 51985 55461 42620 24685 70230 37968 1690 34702 92849 1916 21341 49358 3890 69316 90712 84454 85451 55393 68975 83658 13920 10114 51861 31016 98799 59994 1335 19108 19585 26561 93585 14703 73480 14509 35889 13802 71322 3067 11763 59146 79580 53823 84255 89884 1231 80577 64278 75185 98765 81923 55287 55444 10922 1362 3928 5663 28461 53581 91679 55741 71385 23339 30280 32437 38689 22466 29380 6958 69931 37825 71412 8648 84443 50378 21601 85789 75526 82135 41431 1275 34976 54275 68816 41711 48937 13408 95922 43360 73586 62524 81948 69915 25380 84188 83946 61483 22024 54099 52972 54803 79261 22807 12002 46522 88745 37285 29251 33244 81752 17045 3090 46782 89548 30734 10293 3379 43815 83398 40536 90765 89953 12994 38294 1155 17981 5712 13536 94177 22081 16662 98259 11502 62769 87468 53243 10192 26889 87644 24432 34907 80585 55367 75280 57660 80570 97221 29865 89425 64184 58138 81286 58650 66623 49332 35214 20492 8996 29685 42916 63372 91406 93098 55109 77478 52127 34112 57030 92770 42395 38582 2285 83224 74185 33176 91690 32415 56339 91403 25812 8326 80761 39977 5443 10657 64680 61812 8185 13079 53230 23358 57094 7562 1160 8384 65694 93980 33287 92556 9616 61881 77320 70960 87139 2805 83400 48615 52019 99217 45467 2165 10248 47318 51334 74121 49664 97926 7235 17153 69359 34346 76930 7953 22777 23448 64282 48722 2300 66214 51545 66874 90465 59100 60776 36606 45952 85384 55888 49675 49746 67882 22442 81564 78041 39005 80052 49313 31972 54604 9944 86168 55365 55913 57622 76082 68747 17356 63350 10703 21529 84789 17785 1154 94723 4147 9893 35091 88273 92846 71618 37425 21608 36410 48207 36335 63230 74899 47825 80019 95952 92253 50001 15493 64055 45377 14989 91288 50864 89109 7305 63812 2486 98450 25122 98458 5269 17854 31190 39994 19928 7109 83309 14155 4997 8431 21927 77991 6435 46426 89013 46423 91060 97190 4554 20548 19720 17536 77555 761 90142 28353 12024 49398 14039 52483 77616 65574 98868 51459 62852 40146 88416 59360 81943 25574 74755 79405 49848 46414 57952 63750 37415 15262 23929 782 7524 69337 42135 19170 3631 5911 533 62896 2954 33204 48800 45035 34191 29191 85583 65142 35926 11904 60292 23535 27291 762 32161 91875 37833 32411 65952 21535 43640 68180 89076 10367 99252 947 64369 65411 42713 22304 97464 22163 70546 57575 87741 8601 93700 8495 4720 97790 8079 81309 39880 84232 29031 22409 74045 11832 80392 15907 79297 33462 35057 95771 93436 57231 75674 32705 53706 22227 47124 60130 48674 11343 37334 62736 78253 28738 59024 51490 58917 58877 88258 24990 48125 8904 38850 69662 68593 7025 25879 27173 84976 10049 61751 50759 65267 19397 80659 81891 62129 91225 369 97605 6118 98187 41579 62334 84782 95313 83309 44110 73871 23906 45777 64083 84173 9050 79334 1090 9602 62420 4281 53548 44641 96595 31045 95570 55771 23864 41824 79955 22969 94748 71301 7508 4049 69409 57294 17694 48345 77280 85149 8076 74866 75435 71845 9725 81143 19926 12718 64956 47609 16517 87201 41276 90066 43175 94156 5777 95724 960 58818 28704 54325 82721 66216 48445 92318 65237 1214 34840 65517 3994 8655 62111 67144 4366 4435 83303 12919 55813 97700 78878 16007 69023 52515 18929 75937 49240 10618 68726 80998 56307 47323 74824 87709 95258 78675 92514 25780 84873 49185 36079 63586 32067 68832 88503 31302 12335 95249 71655 7069 69697 54405 55727 25854 30730 35122 33672 58075 94795 76733 80857 59071 98591 57511 80326 87775 94972 81831 31816 62059 19760 29586 95886 52558 46877 67745 30663 88144 53385 25530 78650 19129 29189 96686 66318 55697 65226 78951 94013 84552 63075 28007 9886 49635 30478 24947 96547 33678 21550 65389 33690 749 15821 61012 1490 59054 24042 45445 40232 21581 43526 84260 9580 70266 85995 96651 52771 71244 40065 18082 40854 20099 77711 64370 50948 30320 16902 3226 6429 22104 320 97140 68502 76718 4909 40671 86266 48240 2981 70452 1554 32349 92240 88708 12953 38173 95119 47184 96580 17709 96245 40658 48959 69515 83721 96354 36784 11346 58726 55829 97381 81329 49525 92728 76793 10541 98277 41256 33373 68276 98030 35639 59771 13152 80416 81484 35894 19979 97375 91961 99024 98891 77009 68966 10599 17261 89193 91795 99237 61355 39180 28838 57268 11949 32608 43315 66889 1537 21918 10768 56716 37190 17310 58345 76015 45962 80103 85870 93810 94806 45472 60473 15192 22907 67241 64827 34277 30581 79442 78109 56607 24138 99733 12236 18296 83881 48440 27295 40842 35538 9446 9304 56478 79263 83529 61782 15480 37444 88434 42138 47384 64962 20137 12611 99672 73459 50324 20772 93482 51281 49224 39004 27421 26313 49985 51027 43663 86262 82829 57347 23898 48530 15615 82226 28315 82877 78175 57028 9479 41702 64031 89767 35796 74878 76231 97408 19162 47096 86747 11345 50344 11206 68964 187 62663 20503 66853 73418 90718 65806 58233 16272 76476 56384 91722 83586 50979 39054 80286 68943 12475 27513 12591 25836 24385 20106
45758 48298 18917 85054 59931 25756 50245 19339 49333 36407 59656 93836 30897 6075 82572 18135 68039 57277 9293 16535 67194 65946 15714 60465 39679 83170 61900 43242 69992 87446 81058 36212 77927 86050 28656 71058 42500 60495 46671 26456 35453 28672 83574 95545 60595 80945 61237 35082 4528 20506 3894 17663 20494 15297 31540 37384 98775 52554 96779 25032 82021 18566 91997 63746 23325 85295 85206 62862 86184 46496 14599 17663 9226 28983 65032 5327 87152 98132 10430 60442 48749 40834 54048 13664 30694 29929 44818 65885 59056 3161 34006 57960 8424 40418 47983 88458 91406 28179 4315 61936 78359 57280 63629 80361 64972 74596 10843 23596 74371 98732 88791 68529 66803 10881 21720 70210 54028 52751 68417 27497 72286 94122 82681 16654 70447 13560 39891 79899 72762 84228 14389 51840 97598 24991 30822 13265 47625 55824 50992 1479 48846 36182 76583 13532 2983 52845 43474 89845 5790 37046 54552 92854 24612 72447 47759 5854 45750 54176 20554 19445 63248 12834 97660 63466 20293 99561 83327 31118 85170
