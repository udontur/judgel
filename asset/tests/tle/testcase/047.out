13767215668679
